### PR TITLE
Adds TPL logo, adds alt text, fixes FGT logo

### PR DESCRIPTION
**Pull request by Shrinks99**
- Adds SVG FreeGeek logo to fix current squished version with the wrong dimensions
- Adds TPL logo
- Adds alt text to _all images_
- Adds information about adding alt text in the readme
- Refactors layout, uses flexbox now

### Issues to discuss before merging

- [x] Does Hypha have an SVG available to replace `hypha-wordmark.png`?  I wasn't able to find one in [the Look and Feel directory](https://drive.google.com/drive/folders/1URLZzO0P2IuLjRzvmg_C3ecV0N54GbeJ), would be nice to keep things consistent.
- [x] The TPL logo we were given is not the best one for this space.  [Trajectory's rebrand page](https://trajectoryco.com/work/toronto-public-library/) shows off an initialism mark (also shown below) that is better suited for this format. I've emailed Fiona inquiring about this.  She has responded, I've implemented it and it's been sent to the branding team for final approval.

Update: branding team didn't want the acronym logo, they feel it's not enough to identify the brand in this context, leaving as is.  This PR is ready to go!

### CODE REVIEW

**Review by benhylau — DISMISSED** [review]

File: src/network/supernode1.md [hunk 1]
```diff
@@ -10,48 +10,48 @@ The supernode consists of 2 antennas and a router. The router is connected to th
 
 The router acts as a Babel routing device. The antennas are configured in bridge mode.
 
-![image](images/supernode1-network-layout.png)
+![network diagram consisting of 4 antennas connected to a switch, connected to the internet through Bell's Homehub 2000 router](images/supernode1-network-layout.png)
```
Comment (darkdrgn2k) [inline src/network/supernode1.md]:
```suggestion
![network diagram consisting of 4 antennas connected to a switch, connected to the internet through a bell router](images/supernode1-network-layout.png)
```

**Review by darkdrgn2k — DISMISSED** [review]
Im worried the alt tags are to detailed.  I understand why but flagging that it may not but it may not be sustainable.

**Comment by Shrinks99** [comment]
> Im worried the alt tags are to detailed. I understand why but flagging that it may not but it may not be sustainable.

Added your suggestion, it's always a fine line between accurately describing everything in the photo, keeping things brief, efficiency, and relevance.  I agree that the specific model of router isn't terribly important here.

**Review by TimTor — COMMENTED** [review]
suggest: "dimly well lit" -> "well lit"
Question: "wireless mesh antennas"  - is there a more precise or otherwise preferred wording?
Suggestion: "the door is open, revealing a [Edgerouter] model and [other thing] installed within"

**Review by TimTor — APPROVED** [review]

**Pull request merged**